xmodule/js/spec/video/social_share_spec.js

```diff
@@ -0,0 +1,47 @@
+(function() {
+    'use strict';
+    describe('VideoSocialSharingHandler', function() {
+        var state, spyOpen;
+
+        beforeEach(function() {
+            state = jasmine.initializePlayer('video_all.html');
+            spyOpen = spyOn(window, 'open').and.returnValue(null);
+            window.analytics = jasmine.createSpyObj('analytics', ['track'])
+        });
+
+        afterAll(() => delete window.analytics);
+
+        describe('clicking social share opens the correct URL', function() {
```

Comment:
we won't need to assert on link generation logic here, just that window.open and analytics.track are called with the right values